feature/account/setup/src/main/kotlin/app/k9mail/feature/account/setup/ui/autodiscovery/view/AutoDiscoveryResultHeaderView.kt

```diff
@@ -25,6 +28,12 @@ internal fun AutoDiscoveryResultHeaderView(
 ) {
     Row(
         modifier = Modifier
+            .clickable(
+                indication = null,
+                interactionSource = remember { MutableInteractionSource() }
+            ){
+
+            }
```

Comment:
Whenever you find yourself doing something non-intuitive like "adding a click listener to disable clicks on an view that is normally not clickable", you're likely missing something. In this case it's https://github.com/thunderbird/thunderbird-android/blob/ac19c2765c9ba3d8ac85b4b0170b5cc1361de758/feature/account/setup/src/main/kotlin/app/k9mail/feature/account/setup/ui/autodiscovery/view/AutoDiscoveryResultView.kt#L39

The `Surface` composable that contains `AutoDiscoveryResultHeaderView` is clickable on purpose. Expanding the box will display the server settings if a configuration was found. This functionality needs to be retained.

Instead of overriding the `clickable` modifier on `Surface` via another `clickable` modifier on `AutoDiscoveryResultHeaderView`, the original `clickable` modifier should only be set if a configuration was found.

Reply:
@cketti Done!